compiler/ballerina-lang/src/main/java/io/ballerina/projects/util/ProjectUtils.java

```diff
@@ -734,7 +734,7 @@ public static String getBallerinaPackVersion() {
      * @return ballerina short version
      */
     public static String getBallerinaShortVersion() {
-        try (InputStream inputStream = ProjectUtils.class.getResourceAsStream(ProjectDirConstants.PROPERTIES_FILE)) {
+        try (InputStream inputStream = ProjectUtils.class.getResourceAsStream(PROPERTIES_FILE)) {
             var properties = new Properties();
             properties.load(inputStream);
             return properties.getProperty(ProjectDirConstants.BALLERINA_SHORT_VERSION);
```

Comment:
Noticed that both `BALLERINA_SHORT_VERSION` and `BALLERINA_VERSION` constants point to the same property. Is this intentional?